packages/admin-ui/ui/src/domain/collections/details/index.tsx

```diff
@@ -145,6 +145,17 @@ const CollectionDetails = () => {
           label="Back to Collections"
         />
         <div className="gap-y-xsmall flex flex-col">
+          {getWidgets("product_collection.details.after").map((w, i) => {
+            return (
+              <WidgetContainer
+                key={i}
+                entity={collection}
+                injectionZone="product_collection.details.after"
```

Comment:
thought: Should this not be `before`?
```suggestion
          {getWidgets("product_collection.details.before").map((w, i) => {
            return (
              <WidgetContainer
                key={i}
                entity={collection}
                injectionZone="product_collection.details.before"
```